small-talk/components/censorMess.test.js

```diff
@@ -0,0 +1,20 @@
+import censor from './censorMess.js';
+//ai-gen start (ChatGPT-4, 1)
+// Mock the swearWords module
+jest.mock('./swearWords.js', () => {
+    return ['badword', 'anotherbadword'];
+})
+
+describe('censor function', () => {
+    it('should replace swear words with asterisks', () => {
+        const input = 'This is a badword and anotherbadword in a sentence.';
```

Comment:
Im curious - how does this work if 'badword' and 'anotherbadword' is not in the dictionary of swearWords?